src/components/StockDetails/StockNews.jsx

```diff
@@ -24,17 +35,32 @@ const StockNews = ({ ...otherProps }) => {
             {isStockNewsLoading ? (
                 <LoadingSpinner />
             ) : (
-                <VStack>
-                    {stockNews.map(({ _id, url, title }) => (
-                        <Box key={_id} w="80%">
-                            <NewsArticle
-                                date="2021-01-01"
-                                title={title}
-                                url={url}
-                            />
-                            <Divider my={4} orientation="horizontal" />
-                        </Box>
-                    ))}
+                <VStack align="flex-start" spacing={12} mx={16}>
+                    {stockNews.map(
+                        ({ _id, url, title, urlToImage, publishedAt }) => (
+                            <Link href={url} isExternal w="100%" align="center">
```

Comment:
for the stock news we should also use the onHover: hoverColor, and have it consistent with the news page articles hover

right now the stock news when hovering just underlines the link

Reply:
Tbh this looks cleaner for this sections 
I can change it back if you think otherwise

Reply:
I prefer the onHover effect that we had previously. The link underlining just looks like a regular//default link